src/themes/money/theme.json

```diff
@@ -1778,15 +1781,19 @@
           "backgroundColor": "grey-05",
           "color": "grey-80",
           "fontSize": "md",
+          "pt": ["xxs", ""],
```

Comment:
is this supposed to be an empty string? I think "0" would be better if you dont want any.

Reply:
replaced with 0